packages/desktop-gui/cypress/fixtures/component_specs.json

```diff
@@ -0,0 +1,25 @@
+{
+  "integration": [
```

Comment:
i think we should just deprecate `integrationFolder` and rename it to `e2eFolder`, but not make this breaking changes. We could just display a warning in the cypress gui / stdout telling you to update it.

With that said, we could go ahead and rename it to be officially `E2E Tests` and in the url with `cypress/e2e/<spec>`

Reply:
can it be separate pull request, please